proto/src/internal/error.rs

```diff
@@ -145,3 +146,129 @@ impl PartialEq for OperationError {
         std::mem::discriminant(self) == std::mem::discriminant(other)
     }
 }
+
+impl OperationError {
+    /// This is bad but I don't feel that bad.
+    ///
+    /// It takes something like `CU0001WebauthnAttestationNotTrusted` and turns it
+    /// into `CU0001 - Webauthn Attestation Not Trusted` if it can, otherwise you just get the normal
+    /// debug format with spaces
+    ///
+    /// Probably shouldn't use this with any of the complex types because it'll get weird quick!
+    pub fn variant_as_nice_string(&self) -> String {
```

Comment:
Are we using this anywhere?

Reply:
yeah, aded it [over here](https://github.com/yaleman/kanidm/blob/031acc0bbc43e67774fbce31e6c80a5e1deac8cd/server/core/src/https/errors.rs#L73) as a first run to get a nice response to the user

Reply:
Wouldn't it be better to just have a to_str that matches rather than a regex? Seems a bit overkill ...